webpack.config.dev.js

```diff
@@ -1,30 +1,43 @@
-const path = require("path");
-const { merge } = require("webpack-merge");
-const common = require("./webpack.common");
+const path = require('path');
+const { merge } = require('webpack-merge');
+const SpeedMeasurePlugin = require('speed-measure-webpack-plugin');
+// Interactive UI at port 8888
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+const common = require('./webpack.common');
 
-module.exports = merge(common, {
-  mode: "development",
+const smp = new SpeedMeasurePlugin();
+
+const mergedConfig = merge(common, {
+  mode: 'development',
   output: {
-    path: path.resolve(__dirname, "dev"),
+    path: path.resolve(__dirname, 'dev'),
```

Comment:
In a development setting I don't see the purpose of an output directory.

Reply:
in the files the Components folder uses a capital C in the start. Should it have a lowercase c instead?

Reply:
Okay, so I've researched a bit more on this one and here is the thing:
Webpack-dev-server uses memory to perform hot reloads so it ignores any output folder, however you can still add an output folder so that the bundle analyser tool can read the sizes of the parsed and Gzip bundles. 
However, for the devServer to create and use the output folder, you need to configure it using devMiddleware.
The final configuration could be something like this:

<img width="420" alt="Screenshot 2022-02-23 at 09 23 20" src="https://user-images.githubusercontent.com/75747772/155291517-f87dbefb-2e30-4719-b288-acfab49229c7.png">